clang/lib/DPCT/APIMapping/QueryAPIMapping.cpp

```diff
@@ -7,36 +7,30 @@
 //===----------------------------------------------------------------------===//
 
 #include "QueryAPIMapping.h"
+#include "llvm/Support/raw_ostream.h"
 
 namespace clang {
 namespace dpct {
 
-std::unordered_map<std::string, std::string> APIMapping::EntryMap;
+llvm::DenseMap<llvm::StringRef, llvm::StringRef> APIMapping::EntryMap;
 
-void APIMapping::registerEntry(const std::string &Name,
-                               const std::string &Description) {
-  if (Description == "NA")
-    return;
-  EntryMap[Name] = Description;
+void APIMapping::registerEntry(llvm::StringRef Name,
+                               llvm::StringRef SourceCode) {
+  if (EntryMap.contains(Name)) {
+    llvm::errs() << "The API Mapping cases name is duplicated.\n";
```

Comment:
Line 20 is not required for release build.   how about remove it in release build.

Reply:
done